specs/breadcrumb.md

```diff
@@ -0,0 +1,139 @@
+- [Carbon Design](https://www.carbondesignsystem.com/components/breadcrumb/code/)
+- [FAST Breadcrumb (React)](https://www.npmjs.com/package/@microsoft/fast-components-react-msft)
+- [Lightning Design System](https://www.lightningdesignsystem.com/components/breadcrumbs/#site-main-content)
+- [Material UI](https://material-ui.com/components/breadcrumbs/)
+- [Office Fabric](https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb)
+
+---
+
+### API
+
+*Component Name*
+- `fast-breadcrumb`
+
+*Properties*
+- `slottedBreadcrumbItems` - an HTMLElement[] used in the slotted directive.
+
+*Attribute*
+- aria-current - attribute set on the last breadcrumb item, if it has an `href` set. It will always be `aria-current="page"`.
+
+### Anatomy and Appearance
+
+```html
+<template
+    role="navigation"
+    aria-label="breadcrumb"
+>
+    <div role="list">
+        <slot ${slotted({ property: "slottedBreadcrumbItems", filter: elements() })}></slot>
+    </div>
+</template>
+```
+
+*Slot Names*
+- slotted directive, with a property called `slottedBreadcrumbItems`.
+
+---
+
+## Implementation
+
+```html
+<fast-breadcrumb>
+    <fast-breadcrumb-item name="Home" href="#"></fast-breadcrumb-item>
+    <fast-breadcrumb-item name="Template" href="#"></fast-breadcrumb-item>
+    <fast-breadcrumb-item name="Template Prices" href="#"></fast-breadcrumb-item>
+</fast-breadcrumb>
+```
+
+### Accessibility
+
+The breadcrumb should align to the interaction model provided by the [W3C](https://www.w3.org/TR/wai-aria-practices/#breadcrumb)
+- `aria-current` - W3 specs says when using `aria-current="page"` the element should be a link. `fast-breadcrumb` will set `aria-current="page"` to the last element if there is an href attribute.
+
+### Globalization
+
+The first and last element inside the breadcrumb will be reversed when switching in LTR/RTL.
+
+### Test Plan
+
+While testing is still TBD for our web components, I would expect this to align with the testing strategy and not require any additional test support.
+
+
+
+# Breadcrumb Item
+
+## Overview
+
+The `fast-breadcrumb-item` is placed inside the `fast-breadcrumb` component. It provides a slot with a default anchor. The component also provides a slot for a separator that defaults using a `/`.
+
+### API
+
+*Component Name*
+- `fast-breadcrumb-item`
+
+*Properties*
+- `showSeparator` - is a boolean to show and hide the separator.
+
+*Slots*
+- a slot that has an `<a></a>` as a default control.
+- `separator` - used as part of the visual presentation that signifies the breadcrumb trail. The separator is defaulted to a `/`. To override the default separator, add an element with an attribute of `slot="separator"`.
+
+*CSS Parts*
+- `listitem-container` - class style to align the default slot content and the separator.
+- `separator` - class style to adjust margin layout
+
+### Anatomy and Appearance
+
+```html
+<div
+    role="listitem"
+    class="listitem-container"
+    part="listitem-container"
```

Comment:
Just an FYI to catch this prior to implementation...I think this can just be `listitem`; it doesn't contain the list item, it's serving as the list item.